locations.md

```diff
@@ -108,15 +140,40 @@ Response:
 HTTP/1.1 200 OK
 
 {
-    "type": "zip3",
+    "latitude": 42.3611,
+    "longitude": -71.057,
+    "countryCode": "US",
     "zip3": "021",
-    "name": "021 - Boston, MA",
-    "country": "United States",
-    "serviceArea": 385,
-    "city": "Boston",
-    "state": "MA"
+    "locations": {
+        "data": [
+            {
+                "type": "zip3",
+                "countryCode": "US",
+                "geography": null,
+                "geographyType": "circle",
+                "date": "2022-08-29T20:13:01.000Z",
+                "name": "021 - Boston, MA",
+                "countryId": 3,
+                "state": "MA",
+                "id": 4414,
+                "country": "USA",
+                "display": "021 - Boston, MA",
+                "serviceArea": 385,
+                "city": "Boston",
+                "zip3": "021",
+                "locationId": 4414,
+                "mileage": 0,
+                "extraMileage": 0
+            },
+            ...
+        ],
+        "limit": 11,
+        "offset": 0,
+        "length": 11
+    }
 }
 ```
 #### Note(s)
-1. For international, the response contains a list of locations sorted by a score. The higher the score, the closer that location is to the given coordinates.
-1. You must specify the type of search. (`zip3` or `lat-long`)
\ No newline at end of file
+1. The mileage displays the exact distance between the input coordinates and the locations coordinates, while the extraMileage displays the distance from the input coordinates to the outermost coordinate considered part of the resulting location.
```

Comment:
this should be approximate distance or maybe straight line distance. Really we should just get Miranda to write this sentence for us